build/packages-template/bbb-graphql-middleware/opts-jammy.sh

```diff
@@ -1,3 +1,3 @@
 . ./opts-global.sh
 
-OPTS="$OPTS -t deb -d redis-server,bbb-graphql-server,postgresql"
+OPTS="$OPTS -t deb -d redis-server,bbb-graphql-server,postgresql-17"
```

Comment:
```suggestion
OPTS="$OPTS -t deb -d redis-server,bbb-graphql-server"
```

Only `bbb-graphql-server` to list `postgresql-17` as dependency.